action.yml

```
@@ -87,6 +91,7 @@ runs:
     - --include='${{ inputs.include }}'
     - --exclude='${{ inputs.exclude }}'
     - --apt-packages=${{ inputs.apt_packages }}
+    - --install-commands=${{ inputs.install_commands }}
```

I think you need to add quotes here:

```suggestion
    - --install-commands='${{ inputs.install_commands }}'
```